SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetConnection.cpp

```diff
@@ -79,6 +79,19 @@ void USpatialNetConnection::UpdateLevelVisibility(const FName& PackageName, bool
 	UpdateActorInterest(Cast<AActor>(PlayerController->GetPawn()));
 }
 
+void USpatialNetConnection::FlushDormancy(class AActor* Actor)
```

Comment:
what is the purpose of the elaborated type specifier 'class' here?

Reply:
Ahh good catch, no reason, VA auto-filling my function for me